test/end-to-end.js

```diff
@@ -340,15 +416,10 @@ describe('end-to-end', function() {
       app = require(SANDBOX);
     });
 
-    it('provides "/notes/greet" method', function(done) {
+    it('contains model "message"', function(done) {
       request(app)
-        .get('/api/notes/greet?msg=Tester')
-        .expect(200, function(err, res) {
-          if (err) done(err);
-          expect(res.body)
-            .to.have.property('greeting', 'Hello Tester');
-          done();
-        });
+        .get('/api/messages')
```

Comment:
The functionality described in this test should be preserved, just the endpoint should be different (`GET /api/messages/greet?msg=Tester`).